tests/test_conformer.py

```diff
@@ -79,3 +87,112 @@ def get_output(shape, norm):
+            conv2_channels=64,
+            conv3_channels=64,
+            conv4_channels=32,
+            conv_kernel_size=(3, 3),
+            conv_padding=None,  # =same
+            pool1_stride=(1, 2),  # pool along the feature axis,
+            pool1_kernel_size=(1, 2),
+            pool1_padding=None,
+            pool2_stride=(1, 2),
+            pool2_kernel_size=(1, 2),
+            pool2_padding=None,
+            out_features=256,
+            activation=nn.ReLU(),
+        )
+        conformer_config = ConformerEncoderV1Config(
+            num_layers=8,
+            frontend=ModuleFactoryV1(module_class=VGG4LayerActFrontendV1, cfg=frontend_config),
+            block_cfg=ConformerBlockV1Config(
+                ff_cfg=ConformerPositionwiseFeedForwardV1Config(
+                    input_dim=256,
+                    hidden_dim=256,
+                    dropout=0.2,
+                    activation=nn.functional.silu,
+                ),
+                mhsa_cfg=ConformerMHSAV1Config(
+                    input_dim=256,
+                    num_att_heads=4,
+                    att_weights_dropout=0.2,
+                    dropout=0.2,
+                ),
+                conv_cfg=ConformerConvolutionV1Config(
+                    channels=256, kernel_size=9, dropout=0.2, activation=nn.functional.silu, norm=LayerNormNC(256)
+                ),
+            ),
+        )
+
+        class DummyConformerModel(nn.Module):
+            """ """
+
+            def __init__(self, cfg: conformer_config):
+                super().__init__()
+                self.model = ConformerEncoderV1(cfg=cfg)
+
+            def forward(self, input: torch.Tensor, seq_len: torch.Tensor):
+                seq_mask = tensor_mask_from_length(input, seq_len)
+                logits, seq_mask = self.model(input, seq_mask)
+                return logits, seq_mask
+
+        model = DummyConformerModel(cfg=conformer_config)
+        dummy_data = torch.randn(3, 30, 50)
+        dummy_data_len = torch.IntTensor([30, 20, 15])
+        traced_model = torch.jit.trace(model.eval(), example_inputs=(dummy_data, dummy_data_len))
+
+        dummy_data_len_2 = torch.IntTensor([30, 15, 10])
+
+        outputs_normal, _ = model(dummy_data, dummy_data_len)
+        outputs_traced, _ = traced_model(dummy_data, dummy_data_len)
+        # check tracing results in the same outputs
+        assert torch.allclose(outputs_normal, outputs_traced, atol=1e-5)
+
+        export_onnx(
+            model,
+            (dummy_data, dummy_data_len),
+            f=f,
+            verbose=True,
+            input_names=["data", "data_len"],
+            output_names=["classes"],
+            dynamic_axes={
+                "data": {0: "batch", 1: "time"},
+                "data_len": {0: "batch"},
+                "classes": {0: "batch", 1: "time"},
+            },
+        )
+
+        session = ort.InferenceSession(f.name)
+        outputs_onnx = torch.FloatTensor(
```

Comment:
can you check if forwarding with new data which have a different time dimension works? I have been testing recognition with the exported onnx model which failed at MHSA reshape op. The requested size had a fixed time dim.